src/app/machines/views/MachineDetails/MachineNetwork/BondForm/LACPRateSelect/LACPRateSelect.test.tsx

```diff
@@ -67,40 +62,47 @@ describe("LACPRateSelect", () => {
         label: BondLacpRate.SLOW,
         value: BondLacpRate.SLOW,
       },
-    ]);
+    ];
+
+    for (var i = 0; i < expectedOptions.length; i++) {
```

Comment:
Have you considered this?
```suggestion
    for (let i in expectedOptions) {
```